securedrop/tests/test_i18n.py

```diff
@@ -228,20 +229,6 @@ def test_i18n(journalist_app, config):
         verify_i18n(app)
 
 
-def test_verify_default_locale_en_us_if_not_defined_in_config(config):
```

Comment:
Was going to flag the wholesale removal of a test with no replacement, but I don't think this one actually tests what it's supposed to. I'm OK with deleting it.

Reply:
This behavior is also now auto-validated by mypy since config.TRANSLATION_DIRS has a type of Path (ie. can't be None).